features/yield/FarmingCalculator.tsx

```diff
@@ -141,8 +156,12 @@ const FarmingCalculator = () => {
     if (currentTokenPrice && rollFromPool) {
       setyUSDPrice(currentTokenPrice.toString());
 
+      // Scenario 0: Current token is the roll to token and the roll has occurred, reward to current pool only.
+      if (timeUntilRoll && isRolled && isRollToToken && rollToPool) {
+        setPoolLiquidity(rollToPool.pool.liquidity.toString());
```

Comment:
@chrismaree this fixes the bug where pool liquidity is $0 for the `rollTo` token post-roll